openmetadata-service/src/main/java/org/openmetadata/service/monitoring/EventMonitor.java

```diff
@@ -0,0 +1,28 @@
+package org.openmetadata.service.monitoring;
```

Comment:
```suggestion
/*
 *  Copyright 2022 Collate
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  you may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at
 *  http://www.apache.org/licenses/LICENSE-2.0
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions and
 *  limitations under the License.
 */

package org.openmetadata.service.monitoring;
```

Reply:
Maybe it is missing in other new files too.

Reply:
oh thanks, completely missed it